libsrc/c64/ser/c64-swlink.s

```diff
@@ -45,15 +46,15 @@
 
 ; Jump table
 
-        .word   INSTALL
-        .word   UNINSTALL
-        .word   OPEN
-        .word   CLOSE
-        .word   GET
-        .word   PUT
-        .word   STATUS
-        .word   IOCTL
-        .word   IRQ
+        .word   SWL_INSTALL
+        .word   SWL_UNINSTALL
+        .word   SWL_OPEN
+        .word   SWL_CLOSE
+        .word   SWL_GET
+        .word   SWL_PUT
+        .word   SWL_STATUS
+        .word   SWL_IOCTL
+        .word   SWL_IRQ
```

Comment:
Although of course not technically necessary these names were so far always identical for all drivers of a certain driver "class" (and to https://github.com/cc65/cc65/blob/master/asminc/ser-kernel.inc#L44-L52). I'd really like to keep it that way. Therefore I ask you to change them in all cc65 SER drivers (and https://github.com/cc65/cc65/blob/master/asminc/ser-kernel.inc#L44-L52). As far as I can see SER_INSTALL etc. should work fine.

Reply:
Done. I hope I didn't miss any.

Reply:
Thanks for the adjustment :-)